contracts/integrations/test/framework/assertions/finalizePool.ts

```diff
@@ -0,0 +1,217 @@
+import { WETH9Events, WETH9TransferEventArgs } from '@0x/contracts-erc20';
+import { ReferenceFunctions } from '@0x/contracts-exchange-libs';
+import {
+    AggregatedStats,
+    constants as stakingConstants,
+    PoolStats,
+    StakingEpochFinalizedEventArgs,
+    StakingEvents,
+    StakingRewardsPaidEventArgs,
+} from '@0x/contracts-staking';
+import {
+    assertRoughlyEquals,
+    constants,
+    expect,
+    filterLogsToArguments,
+    toDecimal,
+    verifyEventsFromLogs,
+} from '@0x/contracts-test-utils';
+import { BigNumber } from '@0x/utils';
+
+import { DeploymentManager } from '../deployment_manager';
+import { SimulationEnvironment } from '../simulation';
+
+import { FunctionAssertion, FunctionResult } from './function_assertion';
+
+const PRECISION = 15;
+const COBB_DOUGLAS_ALPHA = toDecimal(stakingConstants.DEFAULT_PARAMS.cobbDouglasAlphaNumerator).dividedBy(
```

Comment:
Is it worth just pulling this in a `before()` block to reduce the chance of it getting out of sync?

Reply:
I think I'll leave as is for now, and add the staking params to the `SimulationEnvironment` instance when I implement the `Governor` fuzz tests